### PR TITLE
Remove lightbulb icon and added night mode to view menu

### PR DESCRIPTION
**Pull request by harianbarasu**

### CODE REVIEW

File: theme.js [hunk 1]
```diff
@@ -41,50 +50,39 @@ define(function(require, exports, module) {
...
-                "tooltip": "Theme",
-                "visible": true
+            menuItem = new ui.item({
+                type: "check",
+                caption: "ToggleTheme",
+                onclick: toggleTheme
             });
 
+            menus.addItemByPath(
+                "View/PresentationDiv", new ui.divider(), 3, plugin
```
Comment (danallan) [inline theme.js]: Should probably be `View/ThemeDiv` for consistency?

File: theme.js [hunk 2]
```diff
@@ -41,50 +50,39 @@ define(function(require, exports, module) {
...
-            imports.ui.insertCss(require("text!./style.css"), options.staticPrefix, plugin);
+            // get the current theme
+            setTheme();
 
-            // style button
-            styleButton();
-
-            // re-style button whenever theme changes
+            // reset global var whenever style changes
             imports.settings.on("user/general/@skin", function(value) {
```
Comment (danallan) [inline theme.js]: This could probably be just: `imports.settings.on("user/general/@skin", setTheme, plugin);`

File: theme.js (hunk 1 above)
Comment (harianbarasu) [inline theme.js]: Looking at it, I'm not sure we even need it because we're not making a new divider, so I'm taking it out.

File: theme.js [hunk 3]
```diff
@@ -41,12 +45,23 @@ define(function(require, exports, module) {
...
+            menuItem = new ui.item({
+                type: "check",
+                caption: "ToggleTheme",
+                onclick: toggleTheme
+            });
+
+            menus.addItemByPath(
+                "View/PresentationDiv", new ui.divider(), 3, plugin
+            );
+            menus.addItemByPath("View/Toggle Theme", menuItem, 4, plugin);
```
Comment (dmalan) [inline theme.js]: Hey, this should be a checkable "Night Mode" above "Presentation Mode", per Asana.

File: theme.js [hunk 4]
```diff
@@ -137,17 +112,6 @@ define(function(require, exports, module) {
...
-            /**
-             * Hides theme button.
-             */
-            hideButton: hideButton,
-
-            /**
-             * Shows theme button.
-             */
-            showButton: showButton,
-        });
```
Comment (kzidane) [inline theme.js:151]: removing these breaks presentation!

Comment (danallan) [inline theme.js:151]: We're removing the theme button altogether, so we should remove this dependency from presentation!

Comment (kzidane) [inline theme.js:151]: @danallan  done per https://github.com/cs50/harvard.cs50.presentation/pull/10!

**Pull request merged**